src/snowflake/snowpark/modin/plugin/_internal/utils.py

```diff
@@ -1491,7 +1493,10 @@ def convert_numpy_pandas_scalar_to_snowpark_literal(value: Any) -> LiteralType:
     if isinstance(value, np.generic):
         value = value.item()
     elif isinstance(value, native_pd.Timestamp):
-        value = value.to_pydatetime()
+        raise ValueError(
+            "Internal error: cannot represent Timestamp as a Snowpark literal. "
```

Comment:
The message is a little confusing here. We cannot represent timestamp so make it a string and then a timestamp?